base/VulkanTools.cpp

```diff
@@ -430,16 +430,15 @@ namespace vks
 			std::ifstream f(filename.c_str());
 			return !f.fail();
 		}
-
+#if !defined(_WIN32)
```

Comment:
Can you elaborate why this is required? Windows is my main platform and I never had any troubles with this.